tests/dragonfly/__init__.py

```diff
@@ -72,7 +72,7 @@ def stop(self, kill=False):
     def _start(self):
         if self.params.existing_port:
             return
-        base_args = [f"--{v}" for v in self.params.args]
+        base_args = ["--use_zset_tree"] + [f"--{v}" for v in self.params.args]
```

Comment:
is this intentionally here?

Reply:
yeah, updated the PR description (too lazy to send a commit only for this)